pymobiledevice3/cli/webinspector.py

```diff
@@ -6,17 +6,27 @@
 import inquirer
 import uvicorn
 from inquirer.themes import GreenPassion
+from prompt_toolkit import PromptSession
+from prompt_toolkit.auto_suggest import AutoSuggestFromHistory
+from prompt_toolkit.history import FileHistory
+from prompt_toolkit.lexers import PygmentsLexer
+from prompt_toolkit.patch_stdout import patch_stdout
+from prompt_toolkit.styles import style_from_pygments_cls
 from pygments import highlight, lexers, formatters
+from pygments.styles import get_style_by_name
 
 from pymobiledevice3.cli.cli_common import Command, wait_return
+from pymobiledevice3.common import get_home_folder
 from pymobiledevice3.exceptions import WirError, InspectorEvaluateError
 from pymobiledevice3.lockdown import LockdownClient
 from pymobiledevice3.services.web_protocol.cdp_server import app
 from pymobiledevice3.services.web_protocol.driver import WebDriver, Cookie, By
 from pymobiledevice3.services.webinspector import WebinspectorService, SAFARI, Application, Page
 
+WEBINSPECTOR_HISTORY_PATH = get_home_folder() / 'webinspector_history'
```

Comment:
This will create the home folder on import

Reply:
move variable inside the function, hopefully i will change it in the future